basis/html/templates/chloe/chloe-tests.factor

```diff
@@ -10,7 +10,7 @@ IN: html.templates.chloe.tests
 
 : run-template ( quot -- string )
     with-string-writer [ "\r\n\t" member? ] reject
-    "?>" split1 nip ; inline
+    "<!DOCTYPE html>" split1 nip ; inline
```

Comment:
This actually makes me want to rethink the second patch. I think that I should actually just drop the prolog entirely, leave it up to the template to include `<!DOCTYPE html>` if they want to, and then modify these templates to explicitly have `<!DOCTYPE html>` in them.

Otherwise, there's a really weird situation wherein these apps, which are voluntarily not using `boilerplate`, *are* in fact getting a boilerplate—the `<!DOCTYPE>`—which they may legitimately not want.

I'll rev this to have that as a solution and we can check in to make sure that looks okay.